brainstormforce/astra
# AST-3591 - Secondary buttons package related issue.

### Description

- Found an issue with Outline buttons backward case.
- Import a starter template (For example: Brandstore).
- Do not set any Button styles from customizer.
- Here, default WordPress Styles will apply to button due to a Gutenberg Backwards Compatibility we had provided in earlier version. see https://d.pr/i/FmamUi
- Switch to package, secondary button default styles applies directly to button instead of WordPress Default Style.

### Screenshots

https://d.pr/i/FmamUi

### Types of changes




Bug Fixes.

### How has this been tested?

- Switch to previous version.
- Import any Starter Template like Brandstore for example.
- Do not apply any button styles from Customizer > Global > Buttons.
- Here, the default WordPress Outline Button Styles should apply.
- Switch to this branch, the default WordPress Styles should apply for Outline and should not break.

### Checklist:
